src/utils.ts

```diff
@@ -69,3 +65,20 @@ export async function ensureDirExistsAndWriteFile(content: string | Buffer, dest
     await mkdir(dirname(dest), options);
     await writeFile(dest, content);
 }
+
+export function getTmpDir() {
+    return mkdtempSync(pathJoin(osTmpdir(), '__vite-svg-2-webfont-'));
+}
+
+export function rmDir(path: string) {
+    fsRm(path, { force: true, recursive: true }, () => {});
+}
+
+export function base64ToArrayBuffer(base64: string) {
+    const binaryString = atob(base64);
```

Comment:
Can we use `Buffer.from(base64, 'base64').toString()`?
```suggestion
    const binaryString = Buffer.from(base64, 'base64').toString();
```

Reply:
Yes, we can, but then it should be
```suggestion
    const stringAsBase64 = Buffer.from(string).toString('base64');
```